typescript/sdk/src/ism/EvmIsmReader.ts

```diff
@@ -214,7 +222,7 @@ export class EvmIsmReader implements IsmReader {
     try {
       const paused = await pausableIsm.paused();
       const owner = await pausableIsm.owner();
-      assert((await pausableIsm.moduleType()) === ModuleType.NULL);
+      this.assertModuleType(await pausableIsm.moduleType(), ModuleType.NULL);
```

Comment:
@paulbalaji - Is this guy redundant because `const paused = await pausableIsm.paused();` will always through an error before this is run?